CHANGELOG.md

```diff
@@ -42,8 +42,9 @@ First version is based on [Warden v0.12.0](https://github.com/davidalger/warden)
 | RabbitMQ | 3.7, 3.8, 3.9, 3.10 |
 | PHP | 7.2, 7.3, 7.4, 8.0, 8.1, 8.2-rc |
 | Node JS | 10, 12, 13, 14, 15, 16, 17, 18 |
+| Elasticsearch | 5, 6, 7.6, 7.7, 7.9, 7.10, 7.12, 7.13, 7.14, 7.15, 7.16, 7.17, 8.0, 8.1, 8.2 |
```

Comment:
I don't think we should duplicate elasticsearch row there

Reply:
Whoops. Messed up the conflict. Thanks for bringing it to my attention!